tests/wpt/metadata/css/css-transitions/pseudo-elements-001.html.ini

```diff
@@ -2,12 +2,6 @@
   [transition padding-left on :before / values]
```

Comment:
What does it mean that this subtest fails?

Reply:
It looks like this test trigger a bug in Layout 2013, where pseudo content that has an empty content string is not represented properly in the flow tree. I've confirmed that the tests pass when these test cases use a non-empty content string.